postmodern/transaction.lisp

```diff
@@ -58,16 +58,22 @@ arguments) to be executed at commit and abort time, respectively."))
      "BEGIN TRANSACTION ISOLATION LEVEL SERIALIZABLE READ WRITE")
     (t "BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED READ WRITE")))
 
+(defun retry-transaction ()
+  (invoke-restart 'retry-transaction))
```

Comment:
Should be:
```lisp
(defun retry-transaction (&optional condition)
  (let ((restart (find-restart 'retry-transaction condition)))
    (when restart (invoke-restart restart))))
```

Reply:
Almost but not quite. Added more descriptive condition signalled.